chef/cookbooks/ohai/files/default/plugins/linux/block_device_extended.rb

```diff
@@ -25,6 +25,21 @@
         File.open("/sys/block/#{dir}/queue/#{check}") { |f| block[dir][check] = f.read_nonblock(1024).strip }
       end
     end
+
+    # correct that the aacraid driver always sets the removable flag
+    if block[dir]["removable"] == "1"
+      path = File.readlink("/sys/block/#{dir}").split("/")
+      # example:
+      #   ../devices/pci0000:00/0000:00:02.0/0000:03:00.0/host0/target0:1:4/0:1:4:0/block/sdb
+      path[0] = "/sys"
```

Comment:
I think with `File.realpath` you could avoid this trick of turning relative path to absolute

Reply:
Done.